### PR TITLE
feat(navbar): Customize Bootstrap navbar

### PR DESCRIPTION
**Pull request by frassinier**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/Talend/bootstrap-theme/blob/master/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
**TODO**

**What is the new behavior?**
**TODO**

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/theme/_navbar.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+///
+$navbar-brand-logo-width: 75px !default;
+
+.navbar {
+    background-color: $brand-primary-darker;
+
+    &-brand {
+        color: #FFF !important;
```
Comment (talend-glorieux) [inline src/theme/_navbar.scss]: Outch `!important`. Is there a way to remove it?

File: src/theme/_navbar.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
...LDR0xN6/lAf124wpMMHwO1QSidex/c1/N/PnrzbSbcHDEufDqxZneKzjSpTOM7mG9Lh0APbjZdvBiDgx2UhTAAi6B7KDhX9Zyr6I4WmN/xN6vNOJn34xTPSll5nyKM3C1BPw4BBBBREnql2KJ9ub3iv4VpEfLlEWfHRRvDf+djooA1h6VTP7d7NpW+G8+yIJdWYlI0d7LaYQoSfo6wzF8c4vNyxBHzY4YVQmjZhKWaHGawXcXZ4dwzPatQIrmIbwBpe95ew8JzGsx/sAM/nXYttvBlK7302nAXsWc4PgG/AKOVt20dnKHvHYRDmh7nCYY4Kdei2EwJ0wrA5xKA9IGzUj5X1VVvsjb/Hwg0HJlivv6oO9UD7oQpnHz4swC4ySZmP1MKxaTUL/ApuI/TDmdznbip7WeqkpUgjzHyTwzHF9bxrAkSueUOTz+25aPCI5lnVPu7XhVw6mrgtxDKe5CPCzBaf/VPh8NwjHO6bF1F/bJcIM4XjOJ5p4jxS2KKC+sEJGi0AX5pdFrAMr/MspMPZmrTfKe9fsJnAKn0+1nO+q0Jqw4kpHEuw7ywKeC887MYp/Q5OX6UPcClrdB/GN0QHqYn5uehMfArI77nssbbfzZxPrR+li9eUcPGdbViqDNasKfez8Hr14Z6vPh+AYyOr4Qjb1VRA2z3E6n7agM83watyqs/6NfISDar/EsM1F4nqHjyg4kxi9wTL0IsjoR5wUNt2Eud4iCCKmPXLiQcd4s8/pvlqzQziBg1NceSHcrTbJgc1+bccE/8wh/LiqzO3Ed/08OUWRG79K/FmW7ReXXRbXV3qffZLuSGfDQ5x73VLlbkOgSY/Jb5IvMIW0bbK8Nzn0iQ//49GWiS4ha2db6SgLJ35jd7y8T3MnttYvd7kM6/uPFu3fDRwF6vNDSpcz72OPOvDuo9DPPiQ4RYH67wTsA2FuOYD2LiHvHHKbidrAatV8Z6EO5z7dhD37R7W5BqU9/311KFYBH0MD6YnlUAgyNQafRQbrhbxG1aEXCAIiLTuo7c4kXTg30ukqwSC7M3o/fKEHKiUrhIIsifo8LZ6gf+G4eg+6SqBIDjSbIzDSwh7uLB0rpOuEgiyKegCgSDjqrtAIBBBFwgEIugCgUAEXSBobfifAAMACmqAMA0+coIAAAAASUVORK5CYII=') no-repeat center left;
```
Comment (talend-glorieux) [inline src/theme/_navbar.scss]: Images on scss?

File: src/theme/_navbar.scss (hunk 1 above)
Comment (frassinier) [inline src/theme/_navbar.scss]: Sure, it is for test purpose, sorry :disappointed:

File: src/theme/_navbar.scss (hunk 2 above)
Comment (jmfrancois) [inline src/theme/_navbar.scss]: this is becaused it's light, it let us to not get a logo file between project, and it's the best default

**Comment by jmfrancois** [comment]
no more ! important here :)

**Pull request merged**